core/src/test/scala/stryker4s/mutants/applymutants/StatementTransformerTest.scala

```diff
@@ -102,28 +101,34 @@ class StatementTransformerTest extends Stryker4sSuite with TreeEquality {
     // Arrange
     val source = "object Foo { def bar: Boolean = 15 >= 4 && 14 < 20 }".parse[Source].get
 
-    val firstOrigTree = source.find(q">=").value
-    val firstMutants = List(EqualTo, GreaterThan, LesserThanEqualTo)
+    val firstOrigTree = source.find(q"15 >= 4").value
+    val firstMutants: Seq[Mutant] = List(EqualTo, GreaterThan, LesserThanEqualTo)
       .map(Mutant(0, firstOrigTree, _))
-    val firstFoundMutant = RegisteredMutant(firstOrigTree, firstMutants)
 
-    val secOrigTree = source.find(q"<").value
-    val secondMutants = List(LesserThanEqualTo, GreaterThan, EqualTo)
+    val secOrigTree = source.find(q"14 < 20").value
+    val secondMutants: Seq[Mutant] = List(LesserThanEqualTo, GreaterThan, EqualTo)
       .map(Mutant(0, secOrigTree, _))
-    val secFoundMutant = RegisteredMutant(secOrigTree, secondMutants)
 
-    val statements = List(firstFoundMutant, secFoundMutant)
+    val statements = firstMutants ++ secondMutants
 
     // Act
     val result = sut.transformSource(source, statements)
 
     // Assert
     result.source should be theSameInstanceAs source
-    val first = result.transformedStatements.head
+
+    val first = findTransformedStatement(result, q"15 >= 4")
     first.originalStatement should equal(q"15 >= 4")
-    first.mutantStatements.map(_.mutated) should contain only (q"15 == 4", q"15 > 4", q"15 <= 4")
-    val sec = result.transformedStatements(1)
-    sec.originalStatement should equal(q"14 < 20")
-    sec.mutantStatements.map(_.mutated) should contain only (q"14 <= 20", q"14 > 20", q"14 == 20")
+    first.mutantStatements should contain theSameElementsAs firstMutants
+
+    val second = findTransformedStatement(result, q"14 < 20")
+    second.originalStatement should equal(q"14 < 20")
+    second.mutantStatements should contain theSameElementsAs secondMutants
+  }
+
+  private[this] def findTransformedStatement(result: SourceTransformations, term: Term): TransformedMutants = {
+    result.transformedStatements
+      .find(mutant => mutant.originalStatement.syntax.equals(term.syntax))
```

Comment:
Well the reason for it was because i didn't know `scala.meta.contrib._` was a thing i was doing `mutant.originalStatement.isEqual(term)` at first but that didn't match. Will fix it